collector/collector.go

```diff
@@ -110,48 +110,24 @@ func (c *Config) MergeConfig(newConfig *Config) error {
 
 func deepMergeSlices(sliceA, sliceB []interface{}) ([]interface{}, error) {
 	// We use the first item in the slice to determine if there are maps present.
```

Comment:
The name of the function and the comment are now misleading. There is no "deep merging" happening anymore.

Reply:
@jimmykarily good point, so I guess that's part of the major change, we will not deep merge anymore at this point ... do you foresee any issues around that?

Reply:
It's just slices that we won't merge. We will still deep merge maps.

Reply:
There will be problems if someone was relying on this (foul) feature. Bad relying on a mistake is a problem of its own :man_shrugging: .

Reply:
yup good point, oki renamed